platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -667,14 +671,13 @@ check_storage_space() {
     check_storage_space_util "$MOUNTPOINT" "$TOTAL_SIZE" "$iotedge_size_buffer"
     ret="$?"
 
-    base_message="IoT Edge requires a minimum storage space of approximately $((container_size + binary_size + iotedge_size_buffer)) MB for installing edge daemon and runtime docker containers. We verified that the the device has $available_storage MB of available storage for File System $(df -P -m "$MOUNTPOINT" | awk '{print $6}')"
+    wrap_debug_message "IoT Edge requires a minimum storage space of $((container_size + binary_size + iotedge_size_buffer)) MB for installing Edge."
+    wrap_debug_message "The device has $available_storage MB of available storage for File System $(df -P -m "$MOUNTPOINT" | awk '{print $6}')"
```

Comment:
I forgot that -m option is not supported in some distros. Can you remove that?

```suggestion
    wrap_debug_message "The device has $available_storage MB of available storage for File System $(df -P  "$MOUNTPOINT" | awk '{print $6}')"
```